build.sh

```diff
@@ -101,9 +101,15 @@ if [ "$xSTATIC_ANALYSIS" = "1" ] ; then
   echo "+ + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + +"
 
   echo "MSGBLD0010: Initializing VERA++ static code analysis."
+  git clone https://github.com/verateam/vera.git
```

Comment:
Vera++ is actually hosted at bitbucket: http://bitbucket.org/verateam/vera . The GitHub version is a clone. Further, cloning from that repo will give you whatever version is at branch master latest. This is not version 1.3.0 (which was [released 2015-01-22](https://bitbucket.org/verateam/vera/wiki/Home)). The last commit on master is also [not stable](https://bitbucket.org/verateam/vera/commits/cf13c413f641b5816a4fc9cd4b2fb248c81bf2bd). You should use the sources from the 1.3.0 download page: https://bitbucket.org/verateam/vera/wiki/Installation